compile_database.py

```diff
@@ -17,6 +17,17 @@
     re.X,
 )
 
+XCODE_BASE_PATH = None
+
+
+def getXcodeBasePath():
+    global XCODE_BASE_PATH
+    if XCODE_BASE_PATH is None:
+        XCODE_BASE_PATH = subprocess.check_output(
```

Comment:
here I don't want to have exception to crash the calling thread. may be a backup to default Path?

Reply:
Good point! I'll add that check later. Thank you for the review!

Reply:
Done!